### ISSUE TITLE
Changing number of blobs

### ISSUE THREAD

**Issue by zmehdiz97** [dave-epstein/blobgan#3]
In appendix E.1 Hyperparameters of your paper you mentioned that you used different model configurations depending on the dataset (number of objects per image). 
I want to train a model with k=40, d_in= 256 and d_style=256. I changed output shape of the layout network according to these values and 'c_model' of the generator to 32 to match to spatial_style number of channels.

This is my config file:

  model:
    name: BlobGAN
    lr: 0.002
    dim: 256
    noise_dim: 512
    resolution: 256
    lambda:
      D_real: 1
      D_fake: 1
      D_R1: 50
      G: 1
      G_path: 2
      G_feature_mean: 10
      G_feature_variance: 10
    discriminator:
      name: StyleGANDiscriminator
      size: ${model.resolution}
    generator:
      name: models.networks.layoutstylegan.LayoutStyleGANGenerator
      style_dim: ${model.dim}
      n_mlp: 8
      size_in: 16
      c_model: 32
      spatial_style: ${model.spatial_style}
      size: ${model.resolution}
    layout_net:
      name: models.networks.layoutnet.LayoutGenerator
      n_features_max: ${model.n_features_max}
      feature_dim: 256
      style_dim: ${model.dim}
      noise_dim: ${model.noise_dim}
      norm_features: true
      mlp_lr_mul: 0.01
      mlp_hidden_dim: 1024
      spatial_style: ${model.spatial_style}
    n_features_min: ${model.n_features}
    n_features_max: ${model.n_features}
    n_features: 40
    spatial_style: true

Hower with this configuration I get very few parameters in the generator:

   | Name                 | Type                                    | Params
0 | discriminator      | StyleGANDiscriminator       | 28.9 M
1 | generator_ema  | LayoutStyleGANGenerator  | 4.2 M
2 | generator           | LayoutStyleGANGenerator | 4.2 M
3 | layout_net_ema  | LayoutGenerator                | 28.6 M
4 | layout_net          | LayoutGenerator                 | 28.6 M

61.7 M    Trainable params
32.9 M    Non-trainable params
94.6 M    Total params
378.321   Total estimated model params size (MB)

**Comment by dave-epstein** [comment]
Try this: `model.layout_net.feature_dim=256 model.dim=256 +model.generator.override_c_in=256`. `feature_dim` and `override_c_in` are for `d_in` in the paper. `model.dim` is for `d_style`.

**Comment by zmehdiz97** [comment]
Thanks. Closing this issue
